src/utils/style.ts

```diff
@@ -107,7 +95,7 @@ export const cssLeadingToken = ({
     }
   | {
       type: Extract<TextType, 'tag'>;
-      size: NoteFontSize;
+      size: TagFontSize;
```

Comment:
The designation was incorrect (the values were the same, so it did not sound like a problem).